content/english/_index.html

```diff
@@ -12,10 +12,80 @@
 {{< button external=true label="Visit the demo" link="https://demo.hedgedoc.org/" outline=true >}}
 {{< /banner >}}
 
+
 {{< features id="corefeatures" >}}
 
-{{< image-promo id="nocodimd" background="primary" title="Wasn't it called CodiMD?" >}}
+
+{{< icon-promo title="Let's get it started!" icon="rocket" background="secondary" right=true >}}
+Installing HedgeDoc is easy! We provide a precompiled build and a docker image. If you need help, then feel free to read our documentation.
+
+{{< button external=true label="Getting started" link="https://github.com/hedgedoc/hedgedoc/#installation--upgrading" >}}
+{{< /icon-promo >}}
+
+
+{{< image-promo id="screenshot" title="Try the demo." icon="child" background="primary" >}}
+Why should you use something you don't know, right? Head over to our demo page and test it out. Try it! It's free!
+Please read the FAQ article about the reliability of the demo.
+
+{{< button external=true label="Visit the demo" link="https://demo.hedgedoc.org/" >}}
+{{< button label="Read the FAQ" link="/faq" >}}
+{{< /image-promo >}}
+
+
+{{< image-promo id="nocodimd" title="Wasn't it called CodiMD?" right=true >}}
 Yes, this project was formerly known as CodiMD, but in 2020 we decided to rename our project to HedgeDoc to avoid confusion with another project of the same name.
 
-{{< button label="Read the full story" link="/history/" outline=true >}}
-{{< /image-promo >}}
\ No newline at end of file
+{{< button label="Read the full story" link="/history/" >}}
+{{< /image-promo >}}
+
+
+{{< icon-promo title="Stay up to date." icon="mastodon" background="secondary" >}}
+We post news and events on a regular base on our social media. Follow us on
+
+{{< button external=true label="Mastodon" link="https://social.hedgedoc.org/" >}}
+{{< /icon-promo >}}
+
+
+{{< icon-promo title="Read the docs." icon="book" background="primary" right=true >}}
+How do I configure OAuth? Can I restrict edits to logged in users?
+Read our documentation or the FAQ and make your HedgeDoc instance fitting perfectly for your use case.
```

Comment:
```suggestion
Read our documentation or the FAQ and make your HedgeDoc instance fit your use case perfectly.
```

Reply:
done

Reply:
shouldn't it be `fits`?

Reply:
No because the verb is `make`. `fit` is used as an adjective